### DISCUSSION TITLE
IS THIS FREE TO USE

### DISCUSSION THREAD

**Discussion by Dobelosepen** [ColorlibHQ/AdminLTE#3791] [Q&A]
IS THIS FREE TO USE? OR I NEED TO BUY IT IF I HOST IT?

**Comment by MGatner** [comment]
It is free under MIT License, though some premium themes are offered for purchase.

**Comment by Dobelosepen** [comment]
Ok thank you for your information

Pada tanggal Jum, 25 Jun 2021 06.09, MGatner ***@***.***>
menulis:
